src/static/js/modules/util/get-breakpoint-state.js

```diff
@@ -0,0 +1,39 @@
+/* ==========================================================================
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var _breakpointsConfig = require( '../../config/breakpoints-config' );
+var _getViewportDimensions = require( './get-viewport-dimensions' )
+                             .getViewportDimensions;
+
+/**
+ * @returns {object} An object literal with Boolean
+ * isMobile, isTablet, isDesktop, isWall, isSmall properties.
+ */
+
+function _inBreakpointRange( breakpointRange, width ) {
+  var min = breakpointRange.min || 0;
+  var max = breakpointRange.max || Number.POSITIVE_INFINITY;
+
+  return min <= width && width <= max;
+}
+
+function getBreakpointState( width ) {
+  var responsiveState = {};
+  width = width || _getViewportDimensions().width;
+
+  for ( var rangeKey in _breakpointsConfig ) { // eslint-disable-line guard-for-in, no-inline-comments, max-len
+    responsiveState['is' + rangeKey[0].toUpperCase() + rangeKey.slice( 1 )] =
+    _inBreakpointRange( _breakpointsConfig[rangeKey], width );
+  }
+
+  responsiveState.isSmall = responsiveState.isMobile ||
```

Comment:
Should we define `isSmall` as a global responsiveState vs just checking for isMobile and isTablet as needed? Thinking there's less ambiguity if we're comparing the ranges we defined in the config themselves vs an abstraction.